_database_api/status.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Status
+layout: default
+---
+
+# Logs
+
+--- row ---
+
+## Get database instances status
+
+--- row ---
+
+`GET https://db-api.scalingo.com/dapi/databases/[:database_id]/instances_status`
+
+{:.table}
+| field | type   | description                              |
+| --    | --     | --                                       |
+| id    | string | Instance Id                              |
+| type  | string | Instance type                            |
+| status| string | Instance status                          |
+| healthy| bool   | Instance healthy or not                 |
+| role  | string | Instance role in the cluster             |
+
+The request will generate will provide information about the instances of a given database
```

Comment:
Can you explain in this paragraph the possible values for the `type`, `status` and `role` attribute? It's unclear just by reading the doc what it could represent